deps/libgit2.version

```diff
@@ -1,2 +1,12 @@
+## jll artifact
+LIBGIT2_JLL_NAME := LibGit2
+
+## source build
 LIBGIT2_BRANCH=v1.4.3
 LIBGIT2_SHA1=465bbf88ea939a965fbcbade72870c61f815e457
+
+## Other deps
+# Specify the version of the Mozilla CA Certificate Store to obtain.
+# The versions of cacert.pem are identified by the date (YYYY-MM-DD) of their changes.
+# See https://curl.haxx.se/docs/caextract.html for more details.
+MOZILLA_CACERT_VERSION := 2022-02-01
```

Comment:
@inkydragon why this is _here_ instead of its own file?

Reply:
https://github.com/JuliaLang/julia/commit/ba0d45d88280223c757ccadb85129f5ed229ff97#diff-077a0e0d8ff036a0ed8b93361a2c26cfc8ce3d96049890896e25c20d06e07461L114-L117

![image](https://user-images.githubusercontent.com/5158738/195988401-e6b80e34-2406-4f33-b846-7e5d35d9b0cd.png)

Because libgit2 depends on it directly.
The download and installation code is also placed in `libgit2.mk`.

CA certificates can be split out as a separate dependency if necessary.